docs/source/developer/baf_minikube_setup.md

```diff
@@ -79,34 +79,39 @@ Before proceeding, first make sure that you've completed [Developer Pre-requisit
 
 1. Choose the DLT platform you want to run and copy the relevant sample network.yaml to build folder; rename it to network.yaml.
 
-   For Fabric, use `platforms/hyperledger-fabric/configuration/samples/network-minikube.yaml`
-
    ```bash
-   cd ~/project/blockchain-automation-framework
-   cp platforms/hyperledger-fabric/configuration/samples/network-minikube.yaml build/network.yaml
+   $ cd ~/project/blockchain-automation-framework
```

Comment:
I dont think $ is necessary, because we tend to copy the complete line, and then we will copy the $